view/frontend/templates/js/replacejs.phtml

```diff
@@ -893,7 +893,8 @@ if (!$block->isOnPageFromWhiteList() && !$block->isMinicartEnabled()) { return;
                             hints = {};
                         } else {
 
-                            var prefill = isObject(data.hints.prefill)
+                            console.log(data);
+                            prefill = isObject(data.hints.prefill)
```

Comment:
Remove `console.log`; add `var` in front of `prefill`, it should stay local to the function scope